.travis.yml

```diff
@@ -15,12 +15,14 @@ node_js:
   - "7.2"
   - "7.3"
   - "7.4"
+  - "7.5"
 install:
   - npm install
   - npm install mongodb
 script:
   - node --stack-trace-limit=1000 --allow-natives-syntax ./tests/unittests.js
   - node --stack-trace-limit=1000 --allow-natives-syntax ./tests/integration.js
+  - eslint .
```

Comment:
`npm run lint`? `eslint` may not be in `$PATH` while npm adds `./node_modules/.bin` to environment of its scripts.